Arduino-Bling/RobotBling/RobotBling.ino

```diff
@@ -95,6 +95,8 @@ enum
   kClimb,
   kVision,
   kEject,
+  kRainbow,
+  kTheater,
```

Comment:
`kRainbow` and `kTheater` are animations and not specific robot states. Given that, why not just change the existing animations (ex. auto_mode, see switch statement) to use these instead of creating new commands? Do you have specific robot state use cases for these animations, if so the enums should reflect that state.